anchor/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "anchor"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anchor-lang = { version = "0.29.0", optional = true}
```

Comment:
I don't see why `anchor-lang` is optional when this crate extends `anchor` specifically

Reply:
in the previous design, everything was in one create. When the feature `include-anchor` was enabled, it included `anchor-lang` related code. In the new design, `anchor` is a separate crate that always depends on `anchor-lang`. Projects that do not use `anchor-lang` will not include our `anchor` create either.

TL;DR: `include-anchor` is an obsolete feature that should be removed